engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-info.html

```diff
@@ -72,7 +72,7 @@
               selector="activationKey.service_level"
               options="serviceLevels()"
               options-format="option for option in options"
-              deletable="true"
+              deletable="activationKey.service_level!=nil && activationKey.service_level!=''"
```

Comment:
Spacing between would be nice between operator and left/right hand sides. Also, there is no such thing as 'nil' in JS.